core/src/main/java/nl/nn/adapterframework/http/rest/ApiListener.java

```diff
@@ -145,7 +145,7 @@ public Message processRequest(String correlationId, Message message, PipeLineSes
 
 	@Override
 	public String getPhysicalDestinationName() {
-		String destinationName = "uriPattern: "+getUriPattern()+"; method: "+getMethod();
+		String destinationName = "uriPattern: /iaf/api/"+getUriPattern()+"; method: "+getMethod();
```

Comment:
Would it be possible to retrieve this string from where it is defined, instead of hard coding it?

Reply:
iaf/api is the framework API, not the ApiServlet endpoint.